pkg/isoeditor/initramfs.go

```diff
@@ -0,0 +1,27 @@
+package isoeditor
+
+import (
+	"io"
+	"os"
+
+	"github.com/openshift/assisted-image-service/pkg/overlay"
+	"github.com/pkg/errors"
+)
+
+func NewInitRamFSStreamReader(irfsPath string, ignitionContent *IgnitionContent) (io.ReadSeeker, error) {
+	irfsReader, err := os.Open(irfsPath)
```

Comment:
This is assuming the initramfs exists locally rather than extracting it from the iso?

Wondering if that fits an "isoeditor" package.

Reply:
Yeah, the plan is to let coreos-installer [do the extraction](https://github.com/openshift/machine-os-images/blob/main/scripts/copy-pxe).
I agree that naming is getting a little awkward This function is very much a mirror of the equivalent ISO one so I feel like they belong in the same package, but I didn't want to go renaming stuff without getting your input first.